cloudbridge/cloud/providers/gce/provider.py

```diff
@@ -73,18 +74,10 @@ def gce_compute(self):
 
     def _connect_gce_compute(self):
         if self.credentials_file:
-            with open(self.credentials_file) as f:
-                data = json.load(f)
-                credentials = SignedJwtAssertionCredentials(
-                    data['client_email'], data['private_key'],
```

Comment:
@baizhang Looks like the self.private_key property is no longer needed and can be required from the constructor altogether?